.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: go
+
+go:
+  - 1.3
+  - 1.4
+  - tip
+
+before_install:
+  # install linting tools
+  - go get github.com/golang/lint/golint
+  - go get golang.org/x/tools/cmd/vet
+  - go get github.com/fzipp/gocyclo
+  # install code coverage tools
+  - go get github.com/axw/gocov/gocov
+  - go get github.com/mattn/goveralls
+  - if ! go get code.google.com/p/go.tools/cmd/cover; then go get golang.org/x/tools/cmd/cover; fi
+
+install:
+  - cd lib/goose
+  - go get -t
+  - go build
+  - cd ../..
```

Comment:
I ported the build script that that was running on [drone.io](https://drone.io/bitbucket.org/liamstask/goose/latest). I would prefer to turn on CI again before we start modifying too much so that we can make sure that we don't break anything.

![screen shot 2015-05-18 at 9 15 29 am](https://cloud.githubusercontent.com/assets/2306786/7681463/760b1f1a-fd3e-11e4-80b9-78c9b82beee5.png)